src/App.js

```diff
@@ -142,7 +142,7 @@ const cards = [
       offlineSpends: 0.015,
       onlineSpends: 0.015,
       rent: 0.0,
-      utilities: 0.015,
+      utilities: 0.0,
```

Comment:
Instead of hardcoding in App.js, you could create a JSON file for each card, that way you know which file to look for and when.